desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/MovementHandlerViewModel.kt

```diff
@@ -146,8 +146,8 @@ class MovementHandlerViewModel {
     fun endDrag(pictureSize: Size) {
         if (!dragPoint.value.toTopLeftOrigin(pictureSize).isInBounds(pictureSize.toRect())
         ) return
-        println(dragPoint.value.toTopLeftOrigin(pictureSize).normalize(pictureSize))
         addClick(dragPoint.value.toTopLeftOrigin(pictureSize).normalize(pictureSize))
+        println(dragPoint.value.toTopLeftOrigin(pictureSize).normalize(pictureSize))
```

Comment:
pls remove this debug statement

Reply:
Removed it
![image](https://github.com/user-attachments/assets/0527f53d-6f96-4d82-9183-f2c2e55f2834)